src/tool/readme.md

```diff
@@ -0,0 +1,27 @@
+# rAthena Tools
+
+## CSV2YAML
+
+This tool will convert older generation TXT (or CSV, comma separated values) databases into the new YAML format. Not every database has been converted to YAML, but it's a process rAthena is taking to make efforts easier for future development.
+
+The rAthena devs have taken steps to produce this tool which can be ran on any environment that supports rAthena, without any extra software!
+
+Once ran, you will be prompted for each database you'd like to convert to YAML, allow any custom features in your collection to be translated over. Please see the `CONVERT_ALL` macro in `src/tools/yaml.hpp` and uncomment before building if you wish to remove the prompts.
```

Comment:
If you added some custom fields to the old format, be sure to modify the conversion code appropriately.